docs/API/functions/Canvas.rst

```diff
@@ -0,0 +1,307 @@
+Canvas
```

Comment:
@embrown is htis automatically generated or did you have to manually create this one?